include/seqan/graph_msa/graph_align_tcoffee_library.h

```diff
@@ -543,23 +542,29 @@ getAlignmentStatistics(String<TFragment, TSpec1> const& matches,
 
     if (length(matches) > 0)
     {
-        TFragIter itFrag = begin(matches, Standard());
-        TFragIter itFragEnd = itFrag;
-        itFrag += from;
-        itFragEnd += to;
+        TFragIter itFrag = begin(matches, Standard()) + from;
+        TFragIter itFragEnd = itFrag + to;
+
```

Comment:
This changes actually had impact, as itFragEnd is now different (as you can see some tests failed). Can you change this line to:
```c++
TFragIter itFragEnd = begin(matches, Standard()) + to;
```

That should restore the correct behaviour.